Signal/src/view controllers/UITests/SignalsViewController.m

```diff
@@ -324,10 +324,8 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 
 - (void)presentThread:(TSThread *)thread keyboardOnViewAppearing:(BOOL)keyboardOnViewAppearing {
     dispatch_async(dispatch_get_main_queue(), ^{
-      if (!_mvc) {
-          _mvc = [[UIStoryboard storyboardWithName:@"Storyboard" bundle:NULL]
-              instantiateViewControllerWithIdentifier:@"MessagesViewController"];
-      }
+      _mvc = [[UIStoryboard storyboardWithName:@"Storyboard" bundle:NULL]
+                instantiateViewControllerWithIdentifier:@"MessagesViewController"];
```

Comment:
Apart from this issue, it's caused a couple others (#1150, #1152), and it offers no measurable value to this release, while introducing the risk of other yet-to-be-discovered issues.

I'm not opposed to reusing the MessageViewController in theory, and see how it paves the way for a split view iPad layout, but we don't have that in this release anyway. If you'd like to submit a PR with your work, it might be useful for a future iPad layout. 

Thanks for looking into it @big-r81.